pkg/controllers/securitypolicy_controller.go

```diff
@@ -189,6 +202,13 @@ func (r *SecurityPolicyReconciler) setupWithManager(mgr ctrl.Manager) error {
 // Start setup manager and launch GC
 func (r *SecurityPolicyReconciler) Start(mgr ctrl.Manager) error {
 	err := r.setupWithManager(mgr)
+	metrics.Register(metrics.ControllerSyncTotal)
+	metrics.Register(metrics.ControllerUpdateTotal)
+	metrics.Register(metrics.ControllerUpdateSuccessTotal)
+	metrics.Register(metrics.ControllerUpdateFailTotal)
+	metrics.Register(metrics.ControllerDeleteTotal)
+	metrics.Register(metrics.ControllerDeleteSuccessTotal)
+	metrics.Register(metrics.ControllerDeleteFailTotal)
 	if err != nil {
```

Comment:
I add a func InitializePrometheusMetrics() and only register metrics for vmc env. 
Could we also register all the metrics in that func?